core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -47,6 +47,22 @@ oppia.factory('SimpleEditorManagerService', [
             x: 0
           }
         }
+      },
+      TextInput: {
+        CUSTOMIZATION_ARGS:  {
+          placeholder: {
+            value: 'replacethis'
+          },
+          rows: {
+            value: 1
+          }
+        },
+        FIRST_ANSWER_GROUP_RULE: {
+          type: 'Contains',
```

Comment:
Should we do a strict Equals? Because an answer like: "Elephants live on Mars" will match "Elephants do not live on Mars" as the former is a subset of the latter.

Reply:
Yes, let's do strict equals for now. We can modify later but I think "Contains" is likely to surprise the creator.

Reply:
Done.